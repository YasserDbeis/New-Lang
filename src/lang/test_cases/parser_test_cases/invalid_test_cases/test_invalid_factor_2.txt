func foo()
{
    x = y + + + + 5;
}