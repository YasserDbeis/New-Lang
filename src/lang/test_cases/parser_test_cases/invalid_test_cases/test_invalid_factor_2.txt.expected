SYNTAX ERROR
Error occurred in Parsing
At token +
Invalid factor
3